src/Compilers/CSharp/Portable/Binder/BindingDiagnosticBag.cs

```diff
@@ -55,7 +56,7 @@ internal static void AddForSymbol(this ref CompoundUseSiteInfo<AssemblySymbol> u
             }
             else
             {
-                Debug.Assert(info.DiagnosticInfo == null || addDependencies); // It would be strange to drop diagnostics, but record dependencies
+                Debug.Assert(info.DiagnosticInfo == null); // It would be strange to drop diagnostics, but record dependencies
```

Comment:
>// It would be strange to drop diagnostics, but record dependencies [](start = 59, length = 67)

Is this comment relevant anymore?

Reply:
> Is this comment relevant anymore?

Yes.

---
In reply to: [362670637](https://github.com/dotnet/roslyn/pull/40640#discussion_r362670637) [](ancestors = 362670637)